docs/Installation/InstallationOnAppleSilicon.md

```diff
@@ -99,36 +97,21 @@ tar -xf blaze-3.8.tar.gz
 mv blaze-3.8 include
 popd
 
-git clone https://github.com/edouarda/brigand.git
-
 # Need master branch of libxsmm to support Apple Silicon
 git clone https://github.com/hfp/libxsmm.git
 pushd libxsmm
 make
 popd
-
-pushd ./src
-git clone https://github.com/Libsharp/libsharp.git
-cd libsharp
-
-# Do not use compiler flag -march=native (unsupported on Apple Silicon)
-sed "s/-march=native//" configure.ac > configure.ac.mod
-mv configure.ac.mod configure.ac
-
-autoupdate
-autoconf
-./configure
-make
-mv auto $SPECTRE_DEPS_ROOT/libsharp
-popd
 ```
 
 Next, clone, patch, and install charm++ v7.0.0.
 ```
 git clone https://github.com/UIUC-PPL/charm
 pushd charm
 git checkout v7.0.0
-./build LIBS multicore-darwin-arm8 --with-production -g3 -j
+git apply $SPECTRE_HOME/support/charm/v7.0.0.patch
+./build LIBS multicore-darwin-arm8 --with-production -g3 -j \
+--without-romio --with-shared
```

Comment:
Isn't the flag `--build-shared`?

Reply:
🤦‍♂️ thanks for catching this! Should have copied and pasted; the actual line I typed was of course --build-shared